install/kubernetes/cilium/templates/cilium-agent/service.yaml

```diff
@@ -44,3 +45,28 @@ spec:
     targetPort: envoy-metrics
 {{- end }}
 {{- end }}
+{{- if and .Values.hubble.enabled .Values.hubble.listenAddress .Values.hubble.peerService.enabled }}
+---
+apiVersion: v1
```

Comment:
@nathanjsweet please put this section entire section in a different file. cilium-cli helm-based can't distinguish multiple yaml specs from one single file.